docs/troubleshooting.rst

```diff
@@ -23,16 +23,38 @@ Here are the most common reasons why a linter does not work:
   to a known syntax (e.g. ``"html"``). The detected syntax is printed to the
   console in debug mode.
   Also note that plugins should move to using the selector setting
-  instead of the old syntaxes attribute. You can use the "selector" linter
+  instead of the old syntaxes attribute. You can use the :ref:`selector` linter
   setting right now instead of the "syntax_map".
 
-- The linter binary is not installed.
+- The linter binary or it dependencies are not installed.
   Be sure to install the linter as documented in the linter plugin’s README.
 
 - The linter binary is installed,
   but its path is not available to SublimeLinter.
   Follow the steps in :ref:`debugging-path-problems` below.
 
+Less common reasons:
```

Comment:
I wouldn't want to add a section like this. For the really uncommon reasons, either a user knows how to debug, or he asks us questions. I'd rather resolve them in the specific issue and perhaps search through the issues when something familiar comes up, than try to maintain a document about it. 

E.g. All plugins that aren't compatible already have issues open for this. Some simply aren't maintained.

If there are conflicts like with gutter icons or popups, either a user already knows basic troubleshooting tactics and figures this out without the docs, or he's simply not going to troubleshoot (or do it wrong).

Reply:
OK, I get the idea.
I suggest replacing it with just ' Anything else should be reported as GitHub issue.'?

Reply:
Yeah, that’s a good idea.